### PR TITLE
adding GUM 

### PR DESCRIPTION
**Pull request by gourmetgalaxy**
adding GUM info and logo

### CODE REVIEW

**Comment by gourmetgalaxy** [comment]
Hi Team,
Our ERC20 token information can be founded here :

https://etherscan.io/token/0x4f5fa8f2d12e5eB780f6082Dd656C565C48E0f24

Thank you.

**Comment by philiparthurmoore** [comment]
BEP20 is here: https://bscscan.com/token/0xc53708664b99df348dd27c3ac0759d2da9c40462. 

This will be on PancakeSwap imminently. See https://github.com/pancakeswap/pancake-frontend/pull/767 and https://twitter.com/PancakeSwap/status/1377264985189978116.

**Comment by NaderOuerdiane** [comment]
I used wrong image dimensions :)) please use this one
logo

File: blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462/info.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Gourmetgalaxy",
+    "website": "https://gourmetgalaxy.io/",
+    "source_code": "https://github.com/gourmetgalaxy",
+    "description": "Gourmet Galaxy is a fusion between a DeFi Yield Farming, a NFTs Gaming dApp, LaunchPad platform aim at solving the problems that overshadowed Yield Farming platforms.",
+    "socials": [
+        {
+            "name": "Twitter",
+            "url": "https://twitter.com/gourmetfinance",
```
Comment (philiparthurmoore) [inline blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462/info.json:9]: I suggest removing the Twitter account until the suspension is removed. This has happened to multiple accounts already and I am not sure when Twitter plans to fix this.

File: blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462/info.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "name": "Gourmetgalaxy",
```
Comment (philiparthurmoore) [inline blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462/info.json:2]: Do you mean `Gourmetgalaxy` or `Gourmet Galaxy`?

File: blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462/info.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            "url": "https://t.me/gourmetgalaxy_chat",
+            "handle": "Gourmetgalaxy"
+        },
+        {
+            "name": "Medium",
+            "url": "https://gourmetgalaxy.medium.com/",
+            "handle": "Gourmetgalaxy"
+        }
+    ],
+    "explorer": "https://bscscan.com/token/0xc53708664b99df348dd27c3ac0759d2da9c40462",
```
Comment (philiparthurmoore) [inline blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462/info.json:23]: Checksum: `0xc53708664b99DF348dd27C3Ac0759d2DA9c40462`.

**Comment by catenocrypt** [comment]
Correction needed: contract folder name should be in 'checksum format', please rename.

```
 XXX   'Expect asset at path /home/runner/work/assets/assets/blockchains/smartchain/assets/0xc53708664b99df348dd27c3ac0759d2da9c40462 in checksum: '0xc53708664b99DF348dd27C3Ac0759d2DA9c40462''
```

**Comment by philiparthurmoore** [comment]
@gourmetgalaxy Have you made all the appropriate changes? I see a commit but don't see an acknowledgment that this is ready for review. Thanks.

**Comment by catenocrypt** [comment]
Sorry, still not in checksum format.  You may need to rename twice, beaacuse lowercase-uppercase only rename may be ignored.

image

**Pull request merged**